labelbox/schema/project.py

```diff
@@ -311,12 +311,21 @@ def _string_from_dict(dictionary: dict, value_with_quotes=False) -> str:
 
         def _validate_datetime(string_date: str) -> bool:
             """helper function validate that datetime is as follows: YYYY-MM-DD for the export"""
+            succeed = False
             if string_date:
```

Comment:
Can we use this approach to check for multiple formats? https://stackoverflow.com/questions/23581128/how-to-format-date-string-via-multiple-formats-in-python

Reply:
That is more concise. Thanks and done.